primitives/src/parachain.rs

```diff
@@ -673,6 +682,8 @@ sp_api::decl_runtime_apis! {
 		/// Extract the abridged head that was set in the extrinsics.
 		fn get_heads(extrinsics: Vec<<Block as BlockT>::Extrinsic>)
 			-> Option<Vec<AbridgedCandidateReceipt>>;
+		/// Get a `SigningContext` with current `SessionIndex` and parent hash.
+		fn signing_context() -> SigningContext<Hash>;
```

Comment:
@bkchr it's safe to add this runtime API?

Reply:
Adding a function is relative safe. Is this already used somewhere? If yes, we should check that the version of the api matches, because we probably have a new node release before a runtime upgrade.